lib/customer.rb

```diff
@@ -0,0 +1,35 @@
+require "csv"
+
+class Customer
+  attr_accessor :email, :address
```

Comment:
`email` and `address` do not need to be `attr_accessor`s.  Instead default to giving the users of your class the minimum access they need.  So these should be readers.